lib/human.py

```diff
@@ -0,0 +1,146 @@
+import io
+import typing
+
+from . import logic, manual_testing
+
+
+# TODO: how should a transpiler handle unexpected values? logging.warning? Throw an exception? Leave a TODO comment in the output?
+
+def main():
+    # path = 'inputs/inputFile.edn'
+    # path = 'inputs/jamestest.edn'
+    path = 'inputs/tqqq_long_term.edn'
+    root_node = manual_testing.get_root_node_from_path(path)
+    print(convert_to_pretty_format(root_node))
+
+
+def pretty_fn(fn_string: str) -> str:
+    if fn_string == logic.ComposerIndicatorFunction.RSI:
+        return "RSI"
+    if fn_string == logic.ComposerIndicatorFunction.CURRENT_PRICE:
+        return "Close"
+    if fn_string == logic.ComposerIndicatorFunction.CUMULATIVE_RETURN:
+        return "CumulativeReturn"
+    if fn_string == logic.ComposerIndicatorFunction.MOVING_AVERAGE_PRICE:
+        return "SMA"
+    print(f"UNEXPECTED function {fn_string}")
+    return fn_string
+
+
+def pretty_comparison(comparator_string: str) -> str:
+    if comparator_string == logic.ComposerComparison.LTE:
+        return "<="
+    if comparator_string == logic.ComposerComparison.LT:
+        return "<"
+    if comparator_string == logic.ComposerComparison.GTE:
+        return ">="
+    if comparator_string == logic.ComposerComparison.GT:
+        return ">"
+    if comparator_string == logic.ComposerComparison.EQ:
+        return "="
+    print(f"UNEXPECTED comparator {comparator_string}")
+    return comparator_string
+
+
+def pretty_indicator(fn: str, val, window_days: typing.Optional[int]):
+    if fn == logic.ComposerIndicatorFunction.CURRENT_PRICE:
+        return f"{pretty_fn(fn)}({val})"
+    else:
+        return f"{pretty_fn(fn)}({val}, {window_days}d)"
```

Comment:
For VectorBT, for example, would have to output something like
```
data["SPY_RSI_30d"] = ta.rsi(data["SPY"], period=30)
```
and then use the same key later to reference it. My point is the variation between transpilers is big enough that I don't want to lock in code structure assumptions beyond "it'll spit out a string in the end."